apps/autostart-stop.html.md.erb

```diff
@@ -0,0 +1,83 @@
+---
+title: Automatically Stop and Start App Machines
+objective: 
+layout: docs
+nav: firecracker
+order: 65
+---
+
+<%= partial "/docs/partials/v2_transition_banner" %>
+
+You can decide whether you want to automatically stop, or downscale, instances of your app when they're idle and automatically restart them when they're needed. Each instance of your app is running on a Fly machine. Fly Proxy controls automatically starting and automaticaly stopping machines according to settings in your `fly.toml` file, in either the `services` or `http_service` sections:
+
+```toml
+...
+[[services]]
+  autostart = true
+  autostop = true
+...
+```
+
+```toml
+...
+[http_service]
+  internal_port = 8080
+  force_https = true
+  auto_stop_machines = true
+  auto_start_machines = true
+...
+```
+
+## Default and recomended values
+
+New V2 apps are automatically started and automatically stopped by default:
+
+```toml
+auto_stop_machines = true
+auto_start_machines = true
+```
+
+Existing apps are automatically started but not automatically stopped by default:
```

Comment:
Truly speaking, Any service without `auto_start(stop)_machines` fields set in fly.toml will use these defaults.
That is the case for existent apps because they don't have them.